webapp/src/Form/Type/TeamClarificationType.php

```diff
@@ -39,7 +39,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('recipient', ChoiceType::class, [
             'choices' => ['Jury' => 'dummy'],
             'attr' => [
-                'readonly' => true,
+                'class' => 'select_readonly',
```

Comment:
The select box is now no longer actually read only. Why is that?

Reply:
Because its not according to the w3c standard (as far as I'm aware)

Reply:
I'd suggest the following fix which makes it w3c compliant while retaining the intended rendering. Just simplifying it by making it  text field that is disabled. (For some reason I'm not able to push to this branch).

```diff
--- a/webapp/src/Form/Type/TeamClarificationType.php
+++ b/webapp/src/Form/Type/TeamClarificationType.php
@@ -8,6 +8,7 @@ use App\Service\DOMJudgeService;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
 
 class TeamClarificationType extends AbstractType
@@ -36,11 +37,9 @@ class TeamClarificationType extends AbstractType
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('recipient', ChoiceType::class, [
-            'choices' => ['Jury' => 'dummy'],
-            'attr' => [
-                'class' => 'select_readonly',
-            ],
+        $builder->add('recipient', TextType::class, [
+            'data' => 'Jury',
+            'disabled' => true,
         ]);
 
         $subjects = [];
```

Reply:
You should have the rights, 
![image](https://user-images.githubusercontent.com/14887731/102993652-da135400-451d-11eb-8edf-3e5227d3c0d5.png)

But I've committed it.